transcom/mymove
# B 21496 int

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21496)

## Summary

This work adds Third address updates to the move history of a move. During implementation, it was discovered that no PPM address updates were being tracked by our move history fetcher, and that address deletion is not being tracked by our move history fetcher. Neither of those are within the scope of this story, so do not expect to see them in the move history.

TEST:

1. Make a new move with all shipment types (non PPM) with second and third addresses
2. Update the addresses
3. As an office user, view the move history of a move and confirm the address changes are present
